dev/io.openliberty.org.jboss.weld5/bnd.bnd

```diff
@@ -18,7 +18,7 @@ javac.target: 11
 
 Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
 
--include= jar:${fileuri;${repo;org.jboss.weld:weld-osgi-bundle;[5.1,5.2)}}!/META-INF/MANIFEST.MF,bnd.overrides
+-include= jar:${fileuri;${repo;org.jboss.weld:weld-osgi-bundle;[5.1,5.2)}}!/META-INF/MANIFEST.MF,check-me-in-if-i-change.bnd
```

Comment:
If this means we're no longer using overrides.bnd, then that file should be deleted.

Reply:
Ah, you have done this. It shows up as a rename which is why I don't see the deletion of overrides.bnd in the file list.